comments/classified.csv

```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
+Here's a very rough first draft of one possible idea,0,0.5
+"On the other hand, this PR itself might not need a release note, if for example the Visual Studio installer will be updated to provide OpenJDK 11 by default in the same release where OpenJDK 11 compatibility ships. In that case, the main topic in the release notes can be the new default OpenJDK version",0,0.5
+"In either case, there will likely be some docs to update, such as: <https://docs.microsoft.com/xamarin/android/troubleshooting/questions/jdk9-errors>",0,0.5
+"@github: How insane would it be to build #code ""separately"" for the Android Designer, so that we can #code out this logic for on-device use but leave it in for the Designer",1,0.5
+"Alternatively, What If™ we moved the logic into a new method, then *removed* that method via the linker when linking apps",0,0.5
+Well at least I deterministically am terrible at spelling,0,0.5
+I'd disagree here: I think there's a beauty to the test classes being self contained. Otherwise we end up with insane inheritance hierarchies that only make what the code is doing _less_ clear. Duplicated code isn't bad; complexity is,0,0.5
+I am curious though why each test has to call .Initialize(),0,0.5
+"@github stupid question, does it run the tree targets all the time? i.e. it would not considered passed until the test passed on the three targets? if so i would keep it to two if we can",1,0.5
+"I don't think the original issue was so terrible. You only ran into this case when you were contextually typed by a type with a string literal constituent to begin with. In practice, this is not frequently encountered",0,0.5
+"However, @github, I think you'll like the the current implementation a lot better. We now only create a string literal type if the constituent types have a string literal type, not just if their content is equal",0,0.5
+"Furthermore, the types are cached in a map of strings to string literal types. Whenever testing the assignability of two string literal types, reference equality kicks in (which is fast)",0,0.5
+"Additionally, error messages are slightly better because you have a specific literal type to report (i.e. #code instead of #code)",0,0.5
+"@github I saw this was done for ObservableItemTemplateCollection for Windows, so I copied that. Mainly just for the dispatcher, but if we pass the view in, then we never have to worry about reaching out into the Application. or Device. worlds",0,0.5
+"Not sure if this is going to do terrible things with lifetime and/or memory. If so, we can change and find another way",1,0.5
+this seems like a terrible idea.  why not just compare to #code #Resolved,0,0.5
+Can we unify these codepaths,0,0.5
+"~I tried, but I couldn't come up with a good solution. I tried:~",0,0.5
+1. ~Take a SyntaxNode from the method to remove from the candidates~,0,0.5
+"~This does not work, because in VB you need go to the parent of the declaration and call #code on it:~",0,0.5
+~#code~,0,0.5
```

Comment:
If there is a really short line like this, you can use the skip button. Maybe remove this entire line:
```suggestion
```